tests/installation/test_executor.py

```diff
@@ -1130,11 +1130,49 @@ def test_executor_should_install_multiple_packages_from_same_git_repository(
     executor._chef = chef
     executor.execute([Install(package_a), Install(package_b)])
 
-    archive_arg = spy.call_args_list[0][0][0]
-    assert archive_arg == tmp_venv.path / "src/demo/subdirectories/package_a"
+    assert spy.call_count == 2
 
-    archive_arg = spy.call_args_list[1][0][0]
-    assert archive_arg == tmp_venv.path / "src/demo/subdirectories/package_b"
```

Comment:
I'd still keep these checks (and just remove the "demo" part).

Reply:
Done.